site/helm-charts.md

```diff
@@ -0,0 +1,513 @@
+
+Similarly, the user configures the "serviceAccount" parameter in values.yaml to specify which service account in the operator's namespace that the operator should use.  If not specified, it defaults to "default" (i.e. the namespace's default service account).  If the user wants to use a different service account, then the user must create the operator's namespace and the service account before installing the operator helm chart.
+
+For example:
+
+```
+helm install kubernetes/charts/weblogic-operator --name my-operator --namespace my-operator-namespace --values values.yaml --wait
+```
+
+This creates a Helm release, named "my-operator" in the "my-operator-namespace" namespace, and creates all of the Kubernetes resources needed to deploy and run the operator, including:
+
+* services for the internal (and external, if configured) REST interfaces
+* cluster roles and bindings for the operator
+* secret containing the private keys for the internal (and external, if configured) REST interfaces
+* ConfigMap containing:
+  * certificates for the internal (and external, if configured) REST interfaces
+  * operator service account information
+  * list of WebLogic domain namespaces which the operator manages
+
+If "my-operator-namespace" exists, it will be used.  If it does not exist, then Helm will create it.
+
+___NOTE: The values.yaml file supplied to the "helm install" command is a user file that contains values that are added to or override the default values defined in the chart.___
+
+You can verify the operator installation by examining the output from the "helm install" command.
+
+## Removing the Operator
+
+The "helm delete" command is used to remove an operator release and its associated resources from the Kubernetes cluster.  The release name used with the "helm delete" command is the same release name used with the "helm install" command (see [Install the Helm Chart](#install-the-helm-chart)).  For example:
+```
+helm delete --purge my-operator
+```
+
+Note: if the operator's namespace did not exist before the Helm Chart was installed, then Helm will create it, however, "helm delete" will not remove it.
+
+## Useful Helm operations
+
+Show the available operator configuration parameters and their default values:
+```
+helm inspect values kubernetes/charts/weblogic-operator
+```
+
+Show the custom values you configured for the operator Helm release:
+```
+helm get values sample-weblogic-operator1
+```
+
+Show all of the values your operator Helm release is using:
+```
+helm get values --all sample-weblogic-operator1
+```
+
+List the Helm releases that have been installed in this Kubernetes cluster:
+```
+helm list
+```
+
+Get the status of the operator Helm release.
+```
+helm status sample-weblogic-operator1
+```
+
+Show the history of the operator Helm release:
+```
+helm history sample-weblogic-operator1
+```
+
+Roll back to a previous operator Helm release, in this case release 1:
```

Comment:
Roll back to a previous version of this operator Helm release, in this case the first version.

Reply:
Done